app/config/services/admin.xml

```diff
@@ -66,6 +66,14 @@
             <argument />
         </service>
 
+        <service id="app.admin.facebook_video" class="AppBundle\Admin\FacebookVideoAdmin">
+            <tag name="sonata.admin" manager-type="orm" label="Contenu » Vidéos Facebook" group="Communication" />
+
+            <argument />
```

Comment:
These arguments are filled by the sonata admin compiler pass, i think there is no need to add comments, furthermore we will need to update the whole configuration file.